backend/src/app/services/site/site.service.ts

```diff
@@ -309,28 +313,52 @@ export class SiteService {
    * @param siteId site Id
    * @returns a single site matching the site ID
    */
-  async findSiteBySiteId(siteId: string, pending: boolean) {
+  async findSiteBySiteId(siteId: string, pending: boolean, userInfo: any) {
```

Comment:
what type is userInfo ? can we use the specific type instead of any ?

Reply:
We don't a have a type for this now. AuthenticatedUser() does not define any specific output type. So we need go with 'any'.